python/feathub/feature_views/derived_feature_view.py

```diff
@@ -80,6 +81,9 @@ def __init__(
                             expression is evaluated after other transformations in the
                             feature view, and only those rows which evaluate to True
                             will be outputted by the feature view.
+        :param keep_source_fields: True iff the metrics defined in the source of this
+                                   feature view should also be reported when this
```

Comment:
Can you update the doc to also make it clear that this will be done recursively?

For example, we can say something like this:

If it is true and this feature view is materialized to a sink, FeatHub will recursively enumerate source feature view of this and every upstream feature views whose keep_source_fields == true, and report metrics defined in those feature views.